index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <nav>
+      <h3>Freecodecamp Survey Form</h3>
+      <style>
+        nav {
+          background-color: black;
+        }
+        .Love button {
+          background-color: grey;
+        }
+        .View Editor {
+          background-color: gray;
+        }
+        .Sign Up {
+          background-color: green;
+        }
+        .Login {
+          background-color: gray;
+        }
+        form {
+          text-align: center;
+        }
+        .submit button {
+          background-color: green;
+        }
+        h1 {
+          text-align: center;
+        }
+        p {
+          text-align: center;
+        }
+      </style>
+      <form action="">
+        <button class="Love Button">Love</button>
+        <button class="View Editor">View in editor</button>
+        <button class="Sign up">Sign up</button>
+        <button class="Login">Login</button>
+      </form>
+    </nav>
+    <h1>Freecodecamp Survey Form</h1>
+    <p>Thank you for taking the time to improve the platform</p>
+    <form>
+      <label for="name">Name</label><br />
+      <input type="text" id="fname" name="fname" /><br />
+      <label for="Email">Email</label><br />
+      <input type="text" id="lname" name="lname" /><br />
+      <label for="age">Age(optional)</label><br />
+      <input type="text" id="fname" name="fname" /><br />
+      <label for="fname">Which option best describes your current role?</label
+      ><br />
+      <input type="text" id="fname" name="fname" /><br />
+
+      <label for="fname">Would you recommend freecodecamp to a friend?</label
+      ><br />
+      <input type="radio" /> <label for="">definately</label><br />
+      <input type="radio" /> <label for="">maybe</label><br />
+      <input type="radio" /> <label for="">not sure</label><br />
+
+      <label for="fname">What is your favorite feature of free codecamp?</label
+      ><br />
+      <input type="textfield" id="fname" name="fname" /><br />
+
+      <label for="fname"
+        >What would you like to see improved? (check all applied)</label
+      ><br />
+
+      <input type="checkbox" /> <label for="name">Frontend projects</label
```

Comment:
Please Include type, id, and name for the Inputs, some of the Inputs don't have these.